src/types/types.ts

```diff
@@ -435,6 +464,17 @@ export interface CollectionNftsResponse {
   pageKey?: string;
 }
 
+/**
+ * The response object for the {@link getSpamContracts} function. The object
+ * contains a list of NFT spam contracts as marked by Alchemy.
+ *
+ * @public
+ */
+export interface GetSpamContractsResponse {
```

Comment:
How do you feel about just returning an array with the addresses (`getSpamContracts(): Promise<string[]>`)?

This way, there's no need to call `.spamContracts` on the returned response. Only reason I can see otherwise is if the NFT API team wants to return an object to add more fields in the future.

Reply:
Sounds good to me!